sentry-samples/sentry-samples-log4j2/src/main/resources/sentry.properties

```diff
@@ -0,0 +1,2 @@
+# not tested yet, not reporting events at all
+in-app-includes="io.sentry.samples"
```

Comment:
@maciejwalkowiak I'd guess solving https://github.com/getsentry/sentry-java/pull/1282/files#r580246878 would solve this one too

Reply:
There's an incorrect DSN in log4j2.xml. Change it to `https://502f25099c204a2fbf4cb16edc5975d1@o447951.ingest.sentry.io/5428563` and events show up.